modules/bench/src/bench.js

```diff
@@ -146,7 +146,7 @@ export default class Bench {
     const opts = {
       ...this.opts,
       multiplier: 1, // multiplier per test case
-      unit: 'iterations',
+      unit: '',
```

Comment:
update doc?

Reply:
Good question. This was done to align with 3.2 behavior. I tested on 3.0-release branch and there is no unit string in the iterations.